src/snmp_errors.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import "github.com/soniah/gosnmp"
+
+func getErrorMessage(snmpErr gosnmp.SNMPError) string {
+	switch snmpErr {
```

Comment:
Just as an opinion I rather using `map[string][string]` instead of `switch` in this case.